### ISSUE TITLE
Kdenlive Credits

### ISSUE THREAD

**Issue by Sprunglesonthehub** [AcreetionOS-Linux/acreetionos-linux.github.io#1]
Slow down the credits for Kdenlive

@Sprunglesonthehub has the resources needed to edit.

**Comment by jajunk** [comment]
Not sure I am racking what this issue is about. What is a Kdenlive Credits?

**Comment by Sprunglesonthehub** [comment]
The video giving credits on the website lol
I need to slow down the animation

**Comment by jajunk** [comment]
I am dumb. I had no idea what you were talking about. LOL. The "Credits" video.

**Comment by Sprunglesonthehub** [comment]
Yeah, exactly. No worries. I could understand how that might be confusing. I'm glad you asked questions to get clarification. I want to make sure we are all on the same page. I would like to make sure there is some form of credit page, section, or similar, on the website, as it's important to give recognition to those who contributed in some form.
